lib/elixir/test/elixir/calendar/iso_test.exs

```diff
@@ -72,7 +70,7 @@ defmodule Calendar.ISOTest do
         Calendar.ISO.day_of_era(2018, 2, 30)
       end
 
-      assert_raise ArgumentError, "invalid date: 2017-11-00", fn ->
```

Comment:
Please revert. If we are already checking the arguments inside, there is no need to do check it in the function clause. It is just more work without any benefits.